src/gsi/stub_wrf_binary_interface.f90

```diff
@@ -29,18 +29,26 @@ end subroutine convert_binary_mass_dummy
 
   subroutine convert_binary_nmm_dummy(this,update_pint,ctph0,stph0,tlm0)
     use kinds, only: r_kind
+    use constants, only: zero 
     implicit none
     class(get_wrf_binary_interface_class), intent(inout) :: this
     logical     ,intent(inout) :: update_pint
     real(r_kind),intent(  out) :: ctph0,stph0,tlm0
+    ctph0 = zero
+    stph0 = zero
+    tlm0 = zero
```

Comment:
This is a stub.  Meaning, this code exists solely for compilation to succeed in making the handshake.
This routine will not be called at runtime, and if it is, then something is wrong elsewhere.
The values `zero` are fine here and possibly serve for suppressing compilation warnings.

Reply:
There are no run-time errors from this stub routine.  They were compilation warnings. NCO wants these warnings to be fixed.  Thanks for your review.